examples/notebooks/introduction_enlyze_python_sdk.ipynb

```diff
@@ -0,0 +1,2981 @@
+       "11  Appliance(uuid=UUID('22a71d1e-8262-4c27-b5fb-6...  \n",
+       "12  Appliance(uuid=UUID('22a71d1e-8262-4c27-b5fb-6...  \n",
+       "13  Appliance(uuid=UUID('22a71d1e-8262-4c27-b5fb-6...  \n",
+       "14  Appliance(uuid=UUID('22a71d1e-8262-4c27-b5fb-6...  \n",
+       "15  Appliance(uuid=UUID('22a71d1e-8262-4c27-b5fb-6...  "
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "melt_pressures_df = (\n",
+    "    variables_df.loc[\n",
+    "        variables_df['display_name'].str.contains(r'Extruder [A-B]')\n",
+    "    ].copy()\n",
+    ")\n",
+    "\n",
+    "melt_pressures_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a3033a2d-b022-4559-8f98-cae16bf42043",
+   "metadata": {},
+   "source": [
+    "## Time series data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03ebad8e-8e54-448f-9d6f-053487ca7d76",
+   "metadata": {},
+   "source": [
+    "Now that we have selected the variables we are interested in, we can use them to get time series data.\n",
+    "\n",
+    "Pandas built-in filtering allows to quickly select the variables we want, but `get_timeseries` expects a list of `enlyze.models.Variable`'s."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "id": "c64809fc-d6dc-4309-9f7a-bd256dc82f67",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "extruder_a_variables = []\n",
+    "\n",
+    "# iterate over all variables\n",
+    "for variable in variables:\n",
+    "    if variable.uuid in extruder_a_variables_df['uuid'].tolist():\n",
+    "        # if a uuid is in the uuid-column of the dataFrame\n",
+    "        # add the variable to the list\n",
+    "        extruder_a_variables.append(variable)\n",
+    "        "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "id": "593d289f-6958-4927-85f1-adecd309fb54",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "melt_pressures = []\n",
+    "\n",
+    "for variable in variables:\n",
+    "    if variable.uuid in melt_pressures_df['uuid'].tolist():\n",
+    "        melt_pressures.append(variable)\n",
+    "        "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ec97ec7-8e4b-4553-ac20-1e668108eabd",
+   "metadata": {},
+   "source": [
+    "### For small periods and quick explorations - get_timeseries()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "66e8c2ce-ceff-43d0-9d98-ceb4b23f2f9e",
+   "metadata": {},
+   "source": [
+    "The `get_timeseries` expects a start and end time, ideally with a timezone specified, and a list of variables.\n",
+    "\n",
+    "It then returns a `TimeseriesData` object, from which the data can be retrieved as a pandas DataFrame or dictionary (records). By default, the column names will be set to the `variable_uuids` but if `use_display_names` is set to True, it will return the dataframe with human-readable names."
```

Comment:
```suggestion
    "It then returns a `TimeseriesData` object, from which the data can be retrieved as a pandas DataFrame or dictionary (records). By default, the column names will be set to the `variable_uuids` but if `use_display_names` is set to True, it will return the DataFrame with human-readable names."
```

Reply:
When advocating for `use_display_names` please mention the caveat that display names are not guaranteed to be unique across an appliance. I'm actually not sure what pandas will do, if there are duplicate column names :thinking:

Reply:
good point, pandas will return both columns then. Isn't this also something worth mentioning in the main documentation?  because if you have that error you probably don't look into this notebook.

Reply:
0e4cca5

Reply:
> Isn't this also something worth mentioning in the main documentation? 

Indeed it is. #34